app/src/main/java/com/rkm/tasky/database/dao/PhotoDao.kt

```diff
@@ -0,0 +1,18 @@
+package com.rkm.tasky.database.dao
+
+import androidx.room.Delete
+import androidx.room.Query
+import androidx.room.Upsert
+import com.rkm.tasky.database.model.PhotoEntity
+
+interface PhotoDao {
+
+    @Query("SELECT * FROM photos WHERE eventId = :id")
+    suspend fun getPhotosByEventId(id: String): List<PhotoEntity>
+
+    @Upsert
+    suspend fun upsertPhotos(photos: List<PhotoEntity>)
+
+    @Delete
+    suspend fun deletePhotos(photos: List<PhotoEntity>)
```

Comment:
Same here, just passing the ID will make your code simpler. You’d then have a `@Query` annotation instead of `@Delete`:

```
@Query(“DELETE FROM photos WHERE eventId = :id”)
suspend fun deletePhoto(id: String)
```